odh/base/monitoring/alertmanager-config.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alertmanager-odh-monitoring
+stringData:
+  alertmanager.yaml: |
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['job']
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 3h
+      receiver: 'github-receiver-issues'
+    receivers:
+    - name: 'webhook'
+      webhook_configs:
+        - url: 'http://alertmanagerwh:30500/'
+    - name: 'github-receiver-issues'
+      webhook_configs:
+        - url: 'http://github-receiver-service.opf-monitoring.svc.cluster.local:9393/v1/receiver'
```

Comment:
Are you planing to add the deployment template for the github receiver in this repo?

Reply:
yes, I have #68 open for that